prisma/seeds/roles.ts

```diff
@@ -7,19 +7,41 @@ export async function seedRoles() {
   await prisma.roles.createMany({
     data: [
       {
-        name: 'Admin',
-        description: 'Responsible for managing a resource'
+        id: "665fdcce-8795-4cec-8c91-0ff1c47e5284",
+        name: "Admin",
+        description: "Responsible for managing a resource",
       },
       {
-        name: 'Train Operator',
-        description: 'Responsible for operating the train'
+        id: "faf787c0-6b97-4dae-84e4-ca947c790bf2",
+        name: "Train Operator",
+        description: "Responsible for operating the train",
```

Comment:
Melhorar a descrição:

`Operates trains, maintains schedules, ensures safety, and communicates with the control center.`